proto/api/v1/vuln_req_service.proto

```diff
@@ -20,22 +20,46 @@ message ListVulnerabilityRequestsResponse {
     repeated storage.VulnerabilityRequest request_infos = 1;
 }
 
+// next available tag: 8
 message DeferVulnRequest {
-    string                             cve                = 1;
-    string                             comment            = 2;
-    storage.VulnerabilityRequest.Scope scope              = 3;
+    enum ExpiryType {
+        TIME            = 0;
```

Comment:
Should we have an `unset` here for compatibility purposes?  Or is `Time` essentially the default?